index.md

```diff
@@ -1,13 +1,116 @@
+| `lvmController.resizer.image.pullPolicy`| Image pull policy for csi-resizer| `IfNotPresent`|
+| `lvmController.resizer.image.tag`| Image tag for csi-resizer| `v1.1.0`|
+| `lvmController.snapshotter.image.registry`| Registry for csi-snapshotter image| `k8s.gcr.io/`|
+| `lvmController.snapshotter.image.repository`| Image repository for csi-snapshotter| `sig-storage/csi-snapshotter`|
+| `lvmController.snapshotter.image.pullPolicy`| Image pull policy for csi-snapshotter| `IfNotPresent`|
+| `lvmController.snapshotter.image.tag`| Image tag for csi-snapshotter| `v4.0.0`|
+| `lvmController.snapshotController.image.registry`| Registry for snapshot-controller image| `k8s.gcr.io/`|
+| `lvmController.snapshotController.image.repository`| Image repository for snapshot-controller| `sig-storage/snapshot-controller`|
+| `lvmController.snapshotController.image.pullPolicy`| Image pull policy for snapshot-controller| `IfNotPresent`|
+| `lvmController.snapshotController.image.tag`| Image tag for snapshot-controller| `v4.0.0`|
+| `lvmController.provisioner.image.registry`| Registry for csi-provisioner image| `k8s.gcr.io/`|
+| `lvmController.provisioner.image.repository`| Image repository for csi-provisioner| `sig-storage/csi-provisioner`|
+| `lvmController.provisioner.image.pullPolicy`| Image pull policy for csi-provisioner| `IfNotPresent`|
+| `lvmController.provisioner.image.tag`| Image tag for csi-provisioner| `v2.1.0`|
+| `lvmController.updateStrategy.type`| Update strategy for lvm localpv controller statefulset | `RollingUpdate` |
+| `lvmController.annotations` | Annotations for lvm localpv controller statefulset metadata| `""`|
+| `lvmController.podAnnotations`| Annotations for lvm localpv controller statefulset's pods metadata | `""`|
+| `lvmController.resources`| Resource and request and limit for lvm localpv controller statefulset containers | `""`|
+| `lvmController.labels`| Labels for lvm localpv controller statefulset metadata | `""`|
+| `lvmController.podLabels`| Appends labels to the lvm localpv controller statefulset pods| `""`|
+| `lvmController.nodeSelector`| Nodeselector for lvm localpv controller statefulset pods| `""`|
+| `lvmController.tolerations` | lvm localpv controller statefulset's pod toleration values | `""`|
+| `lvmController.securityContext` | Seurity context for lvm localpv controller statefulset container | `""`|
+| `rbac.pspEnabled` | Enable PodSecurityPolicy | `false` |
+| `serviceAccount.lvmNode.create` | Create a service account for lvmnode or not| `true`|
+| `serviceAccount.lvmNode.name` | Name for the lvmnode service account| `openebs-lvm-node-sa`|
+| `serviceAccount.lvmController.create` | Create a service account for lvm localpv controller or not| `true`|
+| `serviceAccount.lvmController.name` | Name for the lvm localpv controller service account| `openebs-lvm-controller-sa`|
+| `analytics.enabled` | Enable or Disable google analytics for the controller| `true`|
```

Comment:
lets not have the configurable parameters here, just to avoid them get outdated after each release, instead we can  just point to the actual README.

Reply:
We thought of doing the same in other charts as well.

Reply:
Ok.
I was following https://openebs.github.io/zfs-localpv. Will remove it.

Reply:
done